tests/features/event.feature

```diff
@@ -0,0 +1,260 @@
+@api
+Feature: Event content creation
+  In order to have event on the site
+  As an editor
+  I need to be able to create and see event items
+
+  @javascript
+  Scenario: Make sure that groups of the fields of Event content is configured correctly.
+    Given I am logged in as a user with the "create oe_event content, access content, edit own oe_event content, view published skos concept entities" permission
+    When I visit "the Event creation page"
+    Then I should see "Online"
+    And I should see "Organiser"
+    And I should see "Description"
+    And I should see "Report"
+    And I should see "Registration"
+    And I should not see "Online type"
+    And I should not see "Organiser name"
+    And I should not see "Summary for description"
+    And I should not see "Summary for report"
+    And I should not see "Registration URL"
+
+    When I press "Online"
+    Then I should see "Online type"
+    And I should see "Online time start"
+    And I should see "Online time end"
+    And I should see "Online description"
+    And I should see "Online link"
+    And I press "Online"
```

Comment:
Let's add a comment above the steps that are meant to close back up the field group. Otherwise it's confusing :)